setup.sh

```diff
@@ -13,13 +13,11 @@ bash-completion
 bash-doc
 coreutils
 findutils
-vim
 less
 less-doc
 man-pages
 mandoc
 shadow
-tmux
```

Comment:
Instead of removing programs from the script they should be excludable via a command-line switch. Please correct this behavior otherwise this line will be reverted back to include the program.